lib/API/MP/v1/API.php

```diff
@@ -161,7 +167,8 @@ function unsubscribeFromLists($subscriber_id, array $segments_ids) {
   }
 
   function getLists() {
-    return Segment::whereNotEqual('type', Segment::TYPE_WP_USERS)->findArray();
+    return Segment::whereNotIn('type', [Segment::TYPE_WP_USERS, Segment::TYPE_WC_USERS])
+    ->findArray();
```

Comment:
There is an indent missing here. Or it could be a one-liner...

Reply:
Fixed.